app/models/comment.rb

```diff
@@ -199,6 +154,11 @@ def as_json(_options = {})
 
   def assign_initial_confidence
     self.confidence = self.calculated_confidence
+    # 0 is a placeholder, don't have an id before save.
+    # This will move it up so it gets seen + voted on.
+    self.confidence_order =
```

Comment:
This feels like it could do with more explanation. From my read, we are taking the confidence, normalizing it, then converting it into a big edian 16 bit unsigned integer to use as a sorting key during the CTE.

Reply:
Added in acd4b8fd